File: Backend/FemdAPI.Core/Domains/PersonalPronoun.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace FemdAPI.Core.Domains
{
    public class PersonalPronoun
    {
        [Key]
        public Guid Id { get; protected set; }
        [Required]
        public string Translation { get; protected set; }
        [Required]
        public string FirstPerson { get; protected set; }
        [Required]
        public string SecondPerson { get; protected set; }
        [Required]
        public string ThirdPerson { get; protected set; }
        [Required]
        public string FirstPersonPlural { get; protected set; }
        [Required]
        public string SecondPersonPlural { get; protected set; }
        [Required]
        public string ThirdPersonPlural { get; protected set; }
        [Required]
        public string Formal { get; protected set; }
        [Required]
        public string PhotoPath { get; set; }
        public Guid WordsDictonaryId { get;  set; }
        public WordsDictionary WordsDictionary { get;  set; }

        public PersonalPronoun(string translation, string firstPerson, string secondPerson, string thirdPerson,
            string firstPersonPlural, string secondPersonPlural, string thirdPersonPlural, string formal)
        {
            Translation = translation;
            FirstPerson = firstPerson;
            SecondPerson = secondPerson;
            ThirdPerson = thirdPerson;
            FirstPersonPlural = firstPersonPlural;
            SecondPersonPlural = secondPersonPlural;
            ThirdPersonPlural = thirdPersonPlural;
            Formal = formal;
            Id = Guid.NewGuid();
        }

    }
}

File: Front_beta/FemdBeta/src/app/Models/PersonalPronoun.ts
export class PersonalPronoun
{
  Translation: string;
  FirstPerson: string;
  SecondPerson: string;
  ThirdPerson: string;
  FirstPersonPlural: string;
  SecondPersonPlural: string;
  ThirdPersonPlural: string;
  Formal: string;
  PhotoPath: string;

  constructor(translation: string, firstPerson: string, secondPerson: string, thirdPerson: string,
              firstPersonPlural: string, secondPersonPlural: string, thirdPersonPlural: string,
              formal: string)
  {
    this.Translation = translation;
    this.FirstPerson = firstPerson;
    this.SecondPerson = secondPerson;
    this.ThirdPerson = thirdPerson;
    this.FirstPersonPlural = firstPersonPlural;
    this.SecondPersonPlural = secondPersonPlural;
    this.ThirdPersonPlural = thirdPersonPlural;
    this.Formal = formal;
  }
}
